articles/embedded.md

```diff
@@ -69,15 +69,21 @@ To answer these questions, both the OFERA EU project as well as several others h
 | A-BUILD-ARDUINO | Q-BUILD | Arduino Build | Robotis has explored building all the libraries using the Arduino IDE. This required some manual changes and thus does not scale, but can get you off the ground. | TODO |
 | A-NUTTX-LIBCXX | Q-LANG | C++11/higher support for NuttX | Build libxx from the LLVM project on NuttX, as a pre-requisite to building rclcpp. | TODO |
 | A-PERF-RCLCPP-RESOURCE | Q-PERF | Determine resource use of rclcpp | |  TODO |
-
+| A-DDS-XRCE | Q-MIDDLEWARE | Use of DDS-XRCE standard protocol | This OMG standard defines the protocol used by microcontrollers to publish and subscribe data to a DDS Domain, standard in ROS 2. OFERA and ROBOTIS have proved that it is a suitable protocol to seamless communicate microcontrollers with ROS 2. | [DDS-XRCE](https://www.omg.org/spec/DDS-XRCE/) |
```

Comment:
"proved" sounds very formal. Maybe write "demonstrate" instead. Also "seamless" -> "seamlessly".

btw, I tried adding a commit but it didn't work.